src/widgets/radio.jsx

```diff
@@ -344,6 +357,13 @@ var BaseRadio = React.createClass({
         this.props.onCheckedChange(newChecked);
     },
 
+    noneOfTheAboveIndex: function() {
+        choices = _.map(this.props.choices, (choice) => {
```

Comment:
@Robinson7D i tried `findIndex` first but it raised an error because that was added in a newer version of `underscore` I wasn't sure if it was worth upgrading just for that one method which is why i chose this approach, but i am open to upgrading.